draft-bbs-signatures.md

```diff
@@ -653,14 +653,15 @@ Procedure:
 ### CreateGenerators
 
 The CreateGenerators operation defines how to create a set of generators that form a part of the public parameters used by the BBS Signature scheme to accomplish operations such as Sign, Verify, ProofGen and ProofVerify.
+It is used to compute the set of message generators (H), the signature blinding value generator (H_s), and the signature domain separation tag generator (H_d).
```

Comment:
```suggestion
It is used to compute the set of message generators (`H_1`, `H_2`, etc.,), the signature blinding value generator (`H_s`), and the signature domain generator (`H_d`).
```
I now its confuting but we don't use the generators as part of a list any more for what i know of (in any place that we do is left over by older versions). I think this more accurate in the current notation. Also, we ended up calling the "domain separation tag" just "domain" if I'm not mistaken.

Reply:
Thank you! Then should we also edit the seed named `signature_dst_generator_seed` in the Ciphersuite section?

Reply:
That's a good point! Made the update in #152. Thanks!